package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "lintBoilerplate": "./node_modules/.bin/eslint -c ./templates/boilerplate/client-qx/.eslintrc.json ./templates/boilerplate/client-qx/source/class/qxapp/**/*.js"
+  }
```

Comment:
you are aware of the `qx lint` command ?

Reply:
Is qx linter adding linting capabilities specific to qx on top of eslint or is a replacement?